### PR TITLE
thing: requirements: Simplify readability

### LINKED ISSUE #17: Make Thing Requirements for Linux tutorial easier
Make sure that all steps can be easily followed by users with low to no reading habilities.

- Number all steps.
- Use only absolute paths.
- Remove references and include Zephyr tutorials.
- Fix broken links

### PR DESCRIPTION
**Pull request by alla3**
Add information and change document structure to simplify thing
requirements readability. It resolves #17 . 

Changes:
	- Add Zephyr development environment setup information.
	- Replace relative paths for absolute paths.
	- Number all steps.
	- Remove KNoT Protocol section.
	- Remove heading "$" from code-blocks.
	- Fix syntax.

### CODE REVIEW

File: doc/thing/thing-requirements.rst [hunk 1]
```diff
@@ -12,125 +12,184 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
+   .. code-block:: bash
 
-   $ pip3 install --user west
+      sudo apt install --no-install-recommends git ninja-build gperf \
+           ccache dfu-util device-tree-compiler wget \
+           python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
+           make gcc gcc-multilib
 
-- Create an enclosing folder and change directory
+#. Install dependencies for building OpenThread and KNoT Protocol.
```
Comment (vitbaq) [inline doc/thing/thing-requirements.rst]: Explain that OT and knot protocol are an external project.

File: doc/thing/thing-requirements.rst [hunk 2]
```diff
@@ -12,125 +12,184 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
+
+   .. note:: If you skip this step, it will be necessary to manually source zephyr-env.sh every time a new terminal is opened.
 
 ----------------------------------------------------------------
 
 nRF5x Command Line Tools and Segger JLink
 -----------------------------------------
 
-- Download and extract cli applications at `nRF5 Command Line Tools <https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF5-Command-Line-Tools>`_.
+Download and extract cli applications at `nRF5 Command Line Tools <https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools>`_.
```
Comment (vitbaq) [inline doc/thing/thing-requirements.rst]: Make it clear that you can download and extract using the links or following the below steps.

File: doc/thing/thing-requirements.rst [hunk 3]
```diff
@@ -12,125 +12,184 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
+   .. code-block:: bash
+
+      sudo dpkg -i $HOME/Downloads/nRFCommandLineTools1021tar/nRF-Command-Line-Tools_10_2_1_Linux-amd64.deb
+      sudo dpkg -i $HOME/Downloads/nRFCommandLineTools1021tar/JLink_Linux_V644e_x86_64.deb
 
 ----------------------------------------------------------------
 
-Source KNoT environment configuration file
-------------------------------------------
+Init KNoT SKD Environment
```
Comment (vitbaq) [inline doc/thing/thing-requirements.rst]: Change to Clone and Setup KNoT...

File: doc/thing/thing-requirements.rst [hunk 4]
```diff
@@ -12,125 +12,184 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
 
-   $ source zephyr/zephyr-env.sh
+   .. code-block:: bash
 
-- Program the ``$HOME/.profile`` to always source zephyr-env.sh when you log in.
+      echo "export ZEPHYR_TOOLCHAIN_VARIANT=zephyr" >> $HOME/.profile
+      echo "export ZEPHYR_SDK_INSTALL_DIR=$HOME/zephyr-sdk-0.10.0" >> $HOME/.profile
 
-.. code-block:: bash
+Init Zephyr Environment
```
Comment (vitbaq) [inline doc/thing/thing-requirements.rst]: Change to Clone and Setup Zephyr...

File: doc/thing/thing-requirements.rst [hunk 5]
```diff
@@ -12,125 +12,186 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
+
+#. Extract nRF5 Command Line Tools.
+
+   .. code-block:: bash
 
-- Install nRF5x Command Line and Segger JLink deb package:
+      tar -xvzf $HOME/Downloads/nRFCommandLineTools1021tar.gz -C $HOME/Downloads --one-top-level
 
-.. code-block:: bash
+#. Install nRF5x Command Line and Segger JLink deb package.
```
Comment (jcorde) [inline doc/thing/thing-requirements.rst]: package**s**

File: doc/thing/thing-requirements.rst [hunk 6]
```diff
@@ -12,125 +12,186 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
-.. code-block:: bash
-
-   $ ln -s $KNOT_BASE/scripts/cli.py $HOME/.local/bin/knot
+#. Add cli.py to the path files.
 
-.. note:: This will allow you to call the knot command line interface from any folder.
+   .. code-block:: bash
 
-- Use pip to install cli requirements
+      ln -s $KNOT_BASE/scripts/cli.py $HOME/.local/bin/knot
```
Comment (jcorde) [inline doc/thing/thing-requirements.rst]: Please, use the absolute path.

File: doc/thing/thing-requirements.rst [hunk 7]
```diff
@@ -12,125 +12,186 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
+#. Use pip to install cli requirements.
 
-.. note:: If you skip this step, it will be necessary to manually source knot-env.sh every time a new terminal is opened.
+   .. code-block:: bash
 
-----------------------------------------------------------------
-
-KNoT protocol
--------------
+      pip3 install --user -r $KNOT_BASE/scripts/requirements.txt
```
Comment (jcorde) [inline doc/thing/thing-requirements.rst]: Absolute path too

File: doc/thing/thing-requirements.rst [hunk 8]
```diff
@@ -12,125 +12,186 @@ On other systems, you may consider using the `KNoT Docker <thing-docker.html>`_
...
+#. Add cli.py to the path files.
 
-.. note:: This will allow you to call the knot command line interface from any folder.
+   .. code-block:: bash
 
-- Use pip to install cli requirements
+      ln -s $KNOT_BASE/scripts/cli.py $HOME/.local/bin/knot
 
-.. code-block:: bash
+   .. note:: This will allow you to call the knot command line interface from any folder.
```
Comment (jcorde) [inline doc/thing/thing-requirements.rst]: Please, highlight the "knot" command

**Pull request merged**